<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Print_Codes</class>
 <widget class="QWidget" name="Print_Codes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Печать этикеток с кодами маркировки</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Модель</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbModel"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Размер</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbSize"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblGtin">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblNumber">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Количество этикеток для печати:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spnNumber"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnPrint">
        <property name="text">
         <string>Печать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetup">
        <property name="text">
         <string>Настройка принтера</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
